README.md

```diff
@@ -198,14 +198,14 @@ time, but could be very useful for debugging.
 ## Build and run verification jobs for SMACK
 Leave `aws-c-common` library as is if you have already built verification jobs for SeaHorn.
 
-As a prerequisite, build SMACK following [this guide](https://github.com/smackers/smack/blob/main/docs/installation.md). Note, we require to use [LLVM](http://llvm.org/) [10.0.1](https://releases.llvm.org/download.html#10.0.1). The last commit that SMACK supports LLVM 10 is [4894245](https://github.com/smackers/smack/commit/48942451a1f48e56442c70256f3f20d117f1b309).
+As a prerequisite, build SMACK following [this guide](https://github.com/smackers/smack/blob/main/docs/installation.md). Note, we require to use [LLVM](http://llvm.org/) [14.0.6](https://releases.llvm.org/download.html#14.0.6). The last commit that SMACK supports LLVM 14 is [4894245](https://github.com/smackers/smack/commit/48942451a1f48e56442c70256f3f20d117f1b309).
```

Comment:
Are you sure that the reference to smack commit is current? It seems the same commit as the one that supports llvm10. Probably this paragraph needs to be updated.